### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'group'

### ISSUE THREAD

**Issue by jenniferjang** [raffam/eurosportplayer-dl#2]
I am using the URL https://gb.eurosportplayer.com/event/figure-skating-world-championships/d90aa2f1-37f8-4ad5-8454-22774a887a10 and it's giving me the error 

`AttributeError: 'NoneType' object has no attribute 'group'`

Am I using the wrong URL or can this not be downloaded? Thanks!

**Comment by raffam** [comment]
Can you try to download the new version and run it appending the new --partner parameter to the command line?

**Comment by raffam** [comment]
It works for me without the "--partner" parameter. Can you try again?

**Comment by raffam** [comment]
I assume it has been fixed since it now works for me and this issue has not been updated in a while. Please reopen it if it should not be so

**Comment by jenniferjang** [comment]
> I assume it has been fixed since it now works for me and this issue has not been updated in a while. Please reopen it if it should not be so

Hey, thanks for the update, I didn't get a chance to try it out until now.

I realized it was because my VPN wasn't on (sorry) but then when I turned it on I got this error:

```
jenniferjang-C02R31X5FVH8:eurosportplayer-dl jenniferjang$ python3 eurosportplayer-dl.py --url https://gb.eurosportplayer.com/event/figure-skating-world-championships/d90aa2f1-37f8-4ad5-8454-22774a887a10 --user user --password password
********** STEP 1 **********
********** STEP 2 **********
********** STEP 3 **********
********** STEP 4 **********
********** STEP 5 **********
********** STEP 6 **********
Traceback (most recent call last):
  File "eurosportplayer-dl.py", line 887, in <module>
    main(args)
  File "eurosportplayer-dl.py", line 701, in main
    metadata = get_video_metadata(esp_url, access_token, on_air)
  File "eurosportplayer-dl.py", line 196, in get_video_metadata
    raise Exception("The element rj['data']['Airings'] is empty. Please report this as a bug")
Exception: The element rj['data']['Airings'] is empty. Please report this as a bug
```

Any ideas? Thank you!

**Comment by raffam** [comment]
When I connect to https://gb.eurosportplayer.com/event/figure-skating-world-championships/d90aa2f1-37f8-4ad5-8454-22774a887a10 it says "The content you requested is not available". So unfortunately I am unable to reproduce the issue

**Comment by raffam** [comment]
I can try with my account if you tell the title and the date of the event

**Comment by jenniferjang** [comment]
Ah, let me try again soon. I was actually able to download others, just not that one, but maybe it is because the link changed somehow.
